sourcemash/frontend/static/scss/sourcemash.scss

```diff
@@ -17,7 +17,7 @@
   display: none !important;
 }
 
-.loading {
+.preloader-wrapper {
```

Comment:
Because “hide” was messing with the padding when the user is versus isn’t logged in, so I moved it one div higher.